frontend/trpc/routes/invoices.ts

```
@@ -135,26 +100,41 @@ export const invoicesRouter = createRouter({
     let equityAmountInCents = BigInt(0);
     let equityAmountInOptions = 0;
     let equityPercentage = 0;
-    const { userExternalId, description, totalAmountCents, ...values } = input;
+    const { description, totalAmountCents } = input;
     const dateToday = new Date();
 
     if (ctx.company.equityEnabled) {
       const equityResult = await calculateInvoiceEquity({
         companyContractor: companyWorker,
         serviceAmountCents: Number(totalAmountCents),
         invoiceYear: dateToday.getFullYear(),
-        providedEquityPercentage: values.equityPercentage,
       });
 
       if (!equityResult) {
         throw new TRPCError({
           code: "BAD_REQUEST",
-          message: "Recipient has insufficient unvested equity",
+          message: `Recipient has insufficient unvested equity for their default equity percentage (${companyWorker.equityPercentage}%).`,
         });
       }
```

Added validation to fail fast if the contractor doesn't have sufficient unvested equity for their default percentage. This prevents creating payments that would later fail during processing, giving immediate feedback to the admin.